src/integrationTest/java/org/opensearch/test/framework/cluster/LocalCluster.java

```diff
@@ -413,6 +413,16 @@ public Builder nodeSpecificSettings(int nodeNumber, Map<String, Object> settings
             return this;
         }
 
+        /**
+         * Adds additional plugins to the cluster
+         */
+        @SafeVarargs
+        public final Builder plugin(Class<? extends Plugin>... plugins) {
```

Comment:
we should remove the over-loaded method below.

Reply:
Removed the method that took a single arg and kept this one in.